src/index.ts

```diff
@@ -143,7 +162,8 @@ export class VisualObserver {
     if (this.#elements.has(target)) return;
 
     if (this.#elements.size === 0) {
-      this.#resizeObserver.observe(root);
+      // Note that this will also kick of the resize observer and currently refreshes all elements
```

Comment:
I mean, this only happens when the first element is added, so that's fine?

Reply:
I think so, but I just wanted to note non-obvious behavior!